extensions/amp-story-subscriptions/0.1/test/test-amp-story-subscriptions.js

```diff
@@ -1,14 +1,22 @@
 import '../amp-story-subscriptions';
+import {expect} from 'chai';
```

Comment:
This is globally available, you don't need to import it

Reply:
Removed.